bookish/controllers/Library_Controller.py

```diff
@@ -0,0 +1,29 @@
+from flask import request
```

Comment:
To quote [PEP 8's naming conventions for packages & modules](https://www.python.org/dev/peps/pep-0008/#package-and-module-names):

>     Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability.